docs/topics/gradle/gradle-configure-project.md

```diff
@@ -201,12 +186,6 @@ kotlin {
 
 Note that setting a toolchain via the `kotlin` extension updates the toolchain for Java compile tasks as well.
 
-> To understand which toolchain Gradle uses, run your Gradle build with the [log level `--info`](https://docs.gradle.org/current/userguide/logging.html#sec:choosing_a_log_level)
```

Comment:
Is there any other way to understand it?

Reply:
No, and this section should not be removed

Reply:
Ok, so you still print 
`[KOTLIN] Kotlin compilation 'jdkHome' argument:...`
?

Reply:
Yes as it is related to compiler CLI `jdkHome` argument and not to the removed Gradle one